esmvaltool/recipes/hydrology/recipe_hype.yml

```diff
@@ -0,0 +1,62 @@
+# ESMValTool
+# recipe_pcrglobwb.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim reanalyses files for use in the Hype
+    hydrological model.
+
+
+  authors: ['pelupessy_inti']
+  projects: ['ewatercycle']
+  references: ['acknow_project']
+
+
+datasets:
+  - {dataset: ERA-Interim, project: OBS, tier: 3, type: reanaly, version: 1}
+
+preprocessors:
+  preprocessor: &general
+    extract_shape:
+      shapefile: HYPE_Shapefiles/Meuse_HYPE/Meuse_HYPE.shp
+      method: contains
+      decomposed: true
+    area_statistics:
+      operator: mean
+      
+  daily_temperature:  
+    <<: *general
+    convert_units:
+      units: degC
+    daily_statistics:
+      operator: mean 
+
+  daily_water:
+    <<: *general
+    convert_units:
+      units: mm d-1    
+    daily_statistics:
+        operator: mean 
```

Comment:
We most likely need to use the sum operator instead of the mean.

Reply:
hmm, sum is not accepted...

Reply:
It has been added recently: https://github.com/ESMValGroup/ESMValCore/pull/290